### PR TITLE
Getting started update and overhaul

### PR DESCRIPTION
**Pull request by a-zorina**
Changed Getting Started structure for better accessibility and updated to support the latest changes to Iroha

### CODE REVIEW

**Comment by a-zorina** [comment]
Added fixes to external links and branch, folder, crate and binary names according to the following issues:

- [486](https://github.com/hyperledger/iroha-2-docs/issues/486)
- [499](https://github.com/hyperledger/iroha-2-docs/issues/499)
- [500](https://github.com/hyperledger/iroha-2-docs/issues/500)

A number of outdated folder names in configuration guides are left until the upcoming configuration guide update.

File: src/guide/get-started/launch-iroha.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
+# Launch Iroha
+
+1. Install the prerequisites:
+
+   - [Docker](https://docs.docker.com/get-docker/)
+   - [Docker compose](https://docs.docker.com/compose/)
```
Comment (nxsaken) [inline src/guide/get-started/launch-iroha.md:6]:
```suggestion
   - [Docker Compose](https://docs.docker.com/compose/)
```

File: src/guide/get-started/launch-iroha.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+3. Navigate to the repository root directory.
+ 
+4. Run `docker compose` and specify the network configuration file to bring up a network of 4 containerized peers:
+
+   ```bash
+   $ docker compose -f defaults/docker-compose.yml up
+   ```
+
+   Depending on your set-up, this might either
```
Comment (nxsaken) [inline src/guide/get-started/launch-iroha.md:18]: Would be good to link `iroha_swarm` readme here for setup

File: src/guide/get-started/launch-iroha.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+- To test Iroha code quickly, you can use `docker-compose-single.yml`,
+  which starts a container with a single peer.
+- For testing Iroha code in normal conditions, you can use
+  `docker-compose-local.yml`, which starts 4 connected containers with
+  peers.
+
+::: info
+
+Please note that there is ongoing work to make our configurations for
```
Comment (nxsaken) [inline src/guide/get-started/launch-iroha.md:46]: We already have `iroha_swarm` which allows to generate custom docker compose configs, but it doesn't support individual peer customization yet.

File: src/guide/get-started/operate-iroha-via-cli.md [hunk 4]
```diff
@@ -0,0 +1,359 @@
+# Operate Iroha via CLI
+
+In this tutorial, we will set up and configure the Iroha client CLI, and perform some basic actions on the Iroha blockchain with it.
+
+You can learn how to:
+  - register [domains](#_3-register-a-domain) and [accounts](#_4-register-an-account)
+  - [mint](#_6-register-and-mint-assets), [transfer](#_7-transfer-assets) and [burn](#_8-burn-assets) assets
```
Comment (nxsaken) [inline src/guide/get-started/operate-iroha-via-cli.md:7]: register asset definitions too

File: src/guide/get-started/operate-iroha-via-cli.md [hunk 5]
```diff
@@ -0,0 +1,359 @@
...
+
+```bash
+$ iroha --config path/to/client.toml <COMMAND>
+```
+
+This is not _persistent configuration_: each time you run `iroha`, you must add the `--config path/to/client.toml` command-line argument, unless you put the `client.toml` config file in your working directory.
+
+:::
+
+The account specified in the `[account]` section of `client.toml` is preregistered in the blockchain. Only it can interact with the blockchain for now. If you change the keys or the domain of the account in the config file, you should ensure they are preregistered in the blockchain too.
```
Comment (nxsaken) [inline src/guide/get-started/operate-iroha-via-cli.md:72]: Would be helpful to clarify that this is configured in the default `genesis.json`

File: src/guide/get-started/operate-iroha-via-cli.md [hunk 6]
```diff
@@ -0,0 +1,359 @@
...
+  ```bash
+  $ iroha domain transfer --id="looking_glass" --from "ed0120CE7FA46C9DCE7EA4B125E2E36BDB63EA33073E7590AC92816AE1E861B7048B03@wonderland" --to "ed0120ABA0446CFBD4E12627FFA870FB37993ED83EB1AE0588184B90D832A64C24C379@looking_glass"
+  ```
+2. Check that the ownership changed:
+
+  ```bash
+  $ iroha domain list all
+  ```
+
+3. Switch to the newly created account. For this, we need to modify the `public_key`, `private_key`, and `domain` in the `client.toml` config file with the ones we registered earlier.
```
Comment (nxsaken) [inline src/guide/get-started/operate-iroha-via-cli.md:201]: Maybe it should be clarified that the domain is not necessarily the same as the domain we just transferred, but the domain of the account we want to act as.

File: src/guide/get-started/operate-iroha-via-cli.md [hunk 7]
```diff
@@ -0,0 +1,359 @@
...
+  domain = "looking_glass"
+  public_key = "ed0120ABA0446CFBD4E12627FFA870FB37993ED83EB1AE0588184B90D832A64C24C379"
+  private_key = "802620CBD3D701B561FE98463767729176404DC757D690F78980B8FDD40C171CCB8EB5"
+  ```
+
+::: tip
+[Permissions](/guide/blockchain/permissions) determine accounts rights within Iroha. Domain owners have the most rights in a domain by default, but permission configuration in Iroha is very flexible and can be customized to your needs.
+:::
+
+Now that we control the domain, we can mint assets in it.
```
Comment (nxsaken) [inline src/guide/get-started/operate-iroha-via-cli.md:216]:
```suggestion
Now that we control the domain, we can define and manage assets in it.
```

File: src/guide/get-started/operate-iroha-via-cli.md [hunk 8]
```diff
@@ -0,0 +1,359 @@
...
+
+
+## 6. Register and Mint Assets
+
+In order to mint assets, you need to register the [asset definition](/guide/blockchain/assets) first. We are going to register the _tea_ token within the _looking_glass_ network. To do that, run:
```
Comment (nxsaken) [inline src/guide/get-started/operate-iroha-via-cli.md:221]:
```suggestion
In order to mint an asset, you need to register its [asset definition](/guide/blockchain/assets) first. We are going to register the _tea_ token within the _looking_glass_ network. To do that, run:
```

We should avoid confusion between (fungible) asset definitions and assets. Asset definitions are blueprints, and assets are specific instances owned by accounts. NFTs will likely not have this distinction.

File: .vitepress/config.mts [hunk 9]
```diff
@@ -14,12 +14,12 @@ function nav(): DefaultTheme.NavItem[] {
...
       items: [
         // { text: 'Get Started', link: '/guide/get-started/index' },
-        // { text: 'Build and Install', link: '/guide/get-started/install' },
+        // { text: 'Build and Install', link: '/guide/get-started/install-iroha' },
         // { text: 'Tutorials', link: '/guide/get-started/tutorials'},
         {
           text: 'Get Started',
           items: [
-            { text: 'Build and Install', link: '/guide/get-started/install' },
+            { text: 'Install and Build', link: '/guide/get-started/install-iroha' },
```
[the remark this answers is not in the document]
Comment (a-zorina) [reply .vitepress/config.mts:22]: with short links like these it's usually better, when the link matches the page title. especially if we later end up with some other articles about installing something else.

File: .vitepress/config.mts [hunk 10]
```diff
@@ -93,32 +93,23 @@ function sidebarGuide(): DefaultTheme.SidebarItem[] {
...
-              text: 'Build Iroha Client',
-              link: '/guide/get-started/build',
-            },
-            {
-              text: 'Quick Start with Docker',
-              link: '/guide/get-started/quick-start',
-            },
-          ],
+          text: 'Operate Iroha via CLI',
+          link: '/guide/get-started/operate-iroha-via-cli',
```
[the remark this answers is not in the document]
Comment (a-zorina) [reply .vitepress/config.mts:108]: same as above, I'd leave it as is

File: README.md [hunk 11]
```diff
@@ -1,18 +1,18 @@
...
+- [CLI](https://hyperledger.github.io/iroha-2-docs/guide/operate-iroha-via-cli.html)
 - [Python](https://hyperledger.github.io/iroha-2-docs/guide/python.html)
 - [Rust](https://hyperledger.github.io/iroha-2-docs/guide/rust.html)
 - [Kotlin/Java](https://hyperledger.github.io/iroha-2-docs/guide/kotlin-java.html)
 - [Javascript (TypeScript)](https://hyperledger.github.io/iroha-2-docs/guide/javascript.html)
 
 If you are already familiar with Hyperledger Iroha, we invite you to read about [how Iroha 2 is different](https://hyperledger.github.io/iroha-2-docs/guide/iroha-2.html) from its previous version.
 
-Check the [Hyperledger Iroha 2](https://github.com/hyperledger/iroha/tree/iroha2-dev#hyperledger-iroha) repository for more detailed information about API and available features.
+Check the [Hyperledger Iroha 2](https://github.com/hyperledger/iroha/) repository for more detailed information about API and available features.
```
[the remark this answers is not in the document]
Comment (a-zorina) [reply README.md]: fixed in commit fec265d9a98b1d356133521d2320837db466d62f

File: src/guide/configure/client-configuration.md [hunk 12]
```diff
@@ -18,7 +18,7 @@ $ kagami config client > client-config.json
 
 ## Public and Private Keys
 
-The `configs/client_cli/config.json` client configuration file should contain a pair of the user's public `PUBLIC_KEY` and private `PRIVATE_KEY` cryptographic keys for their account's `ACCOUNT_ID`.
+The `defaults/client.toml` client configuration file should contain a pair of the user's public `PUBLIC_KEY` and private `PRIVATE_KEY` cryptographic keys for their account's `ACCOUNT_ID`.
```
[the remark this answers is not in the document]
Comment (a-zorina) [reply src/guide/configure/client-configuration.md]: fixed in commit fec265d9a98b1d356133521d2320837db466d62f

File: src/guide/get-started/index.md [hunk 13]
```diff
@@ -1,35 +1,61 @@
...
+can:
+
+- Create and manage custom
+  [fungible assets](/reference/glossary.md#fungible-assets), such as currencies,
+  gold, and others
+- Create and manage [non-fungible](/reference/glossary.md#non-fungible-assets)
+  assets
+- Manage user accounts with a domain hierarchy and multi-signature
+  transactions
+- Use efficient portable smart contracts implemented either via
```
[the remark this answers is not in the document]
Comment (a-zorina) [reply src/guide/get-started/index.md]: fixed in commit fec265d9a98b1d356133521d2320837db466d62f

File: src/guide/get-started/operate-iroha-via-cli.md (hunk 4 above)
Comment (a-zorina) [reply src/guide/get-started/operate-iroha-via-cli.md]: fixed in commit fec265d9a98b1d356133521d2320837db466d62f

File: src/guide/get-started/operate-iroha-via-cli.md (hunk 5 above)
Comment (a-zorina) [reply src/guide/get-started/operate-iroha-via-cli.md]: fixed in commit fec265d9a98b1d356133521d2320837db466d62f

File: src/guide/get-started/operate-iroha-via-cli.md (hunk 6 above)
Comment (a-zorina) [reply src/guide/get-started/operate-iroha-via-cli.md]: fixed in commit fec265d9a98b1d356133521d2320837db466d62f

File: src/guide/get-started/operate-iroha-via-cli.md (hunk 7 above)
Comment (a-zorina) [reply src/guide/get-started/operate-iroha-via-cli.md]: fixed in commit fec265d9a98b1d356133521d2320837db466d62f

File: src/guide/get-started/operate-iroha-via-cli.md (hunk 8 above)
Comment (a-zorina) [reply src/guide/get-started/operate-iroha-via-cli.md]: fixed in commit fec265d9a98b1d356133521d2320837db466d62f

File: src/guide/get-started/index.md [hunk 14]
```diff
@@ -1,35 +1,60 @@
...
+  transactions
+- Use efficient portable smart contracts implemented with 
+  [WebAssembly](/guide/blockchain/wasm.md) and [Iroha Special Instructions](/guide/blockchain/instructions.md)
+- Use both permissioned and permission-less blockchain deployments
+
+
+
+## Get Started
+
+1. [Install Iroha 2](/guide/get-started/install-iroha.md).
```
Comment (0x009922) [inline src/guide/get-started/index.md:22]: In some places it we now use just "Iroha", without 2. Should it be here too?

**Review by yamkovoy — APPROVED** [review]

File: src/guide/blockchain/wasm.md [hunk 15]
```diff
@@ -70,7 +70,7 @@ edition = "2021"
 crate-type = ['cdylib']
 
 [dependencies]
-iroha_wasm = { git = "https://github.com/hyperledger/iroha/", branch = "iroha2-dev" }
+iroha_wasm = { git = "https://github.com/hyperledger/iroha/"}
```
Comment (nxsaken) [inline src/guide/blockchain/wasm.md:73]:
```suggestion
iroha_wasm = { git = "https://github.com/hyperledger/iroha/" }
```

File: src/guide/get-started/install-iroha-2.md [hunk 16]
```diff
@@ -0,0 +1,75 @@
...
+
+   ```bash
+   $ mkdir -p ~/Git
+   ```
+
+   ::: tip
+
+   On macOS, if you get the `fatal: could not create work tree dir 'iroha': Read-only file system` error, that is because the home directory is not a real file system.
+
+   To fix this, create a directory named `Git`.
```
Comment (nxsaken) [inline src/guide/get-started/install-iroha-2.md:45]: `Git` is not a special name, maybe rephrase this to make it clearer? There has been slight confusion over this: https://github.com/hyperledger/iroha-2-docs/pull/479

File: src/guide/iroha-2.md [hunk 17]
```diff
@@ -1,140 +1,84 @@
...
-Iroha 2 is a **complete** re-write of
-[Hyperledger Iroha](https://iroha.readthedocs.io/en/develop/index.html) in
-Rust. As of writing, the two projects are developed concurrently.
+<!-- TODO: consider rewriting completely to accommodate the style change -->
 
-In this introduction we outline the differences between the two versions of
-Iroha and highlight the new features of Iroha 2. It should be of particular
-interest to those who are already familiar with Iroha but wish to upgrade
-and start using the newer version.
+Iroha 2 is a **complete** rewrite of [Hyperledger Iroha](https://iroha.readthedocs.io/en/develop/index.html) in Rust. As of writing, the two projects are developed concurrently.
```
Comment (nxsaken) [inline src/guide/iroha-2.md:5]: Iroha 1 isn't being actively developed anymore

**Review by nxsaken — COMMENTED** [review]
Minor suggestions, otherwise lgtm

File: src/guide/iroha-2.md (hunk 17 above)
Comment (a-zorina) [reply src/guide/iroha-2.md]: fixed in commit 68ae813491f1d7b0c5bdaa8350b44d17772179f8

File: src/guide/get-started/install-iroha-2.md (hunk 16 above)
Comment (a-zorina) [reply src/guide/get-started/install-iroha-2.md:45]: fixed in commit 68ae813491f1d7b0c5bdaa8350b44d17772179f8

**Review by nxsaken — APPROVED** [review]

File: src/guide/get-started/launch-iroha.md (hunk 2 above)
Comment (a-zorina) [reply src/guide/get-started/launch-iroha.md]: linked it in the Docker Options section a bit lower

File: src/guide/get-started/launch-iroha.md (hunk 3 above)
Comment (a-zorina) [reply src/guide/get-started/launch-iroha.md]: linked the iroha_swarm readme here
